src/indexing/EventIndex.js

```diff
@@ -170,7 +173,12 @@ export default class EventIndex {
             return;
         }
 
-        const e = ev.toJSON().decrypted;
+        const jsonEvent = ev.toJSON();
+
+        let e;
+        if (ev.isEncrypted()) e = jsonEvent.decrypted;
+        else e = jsonEvent;
```

Comment:
could shorten this to just `const e = ev.isEncrypted() ? jsonEvent.decrypted : jsonEvent;`